### PR TITLE
Add periodic baremetal e2e ipv4 ovn job

### PR DESCRIPTION
**Pull request by jluhrsen**
Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.11.yaml [hunk 1]
```diff
@@ -230,6 +230,11 @@ tests:
       BASE_DOMAIN: ci2.azure.devcluster.openshift.com
       TEST_TYPE: upgrade-conformance
     workflow: openshift-upgrade-azure-single-node
+- as: e2e-metal-ipi-ovn-ipv4
+  interval: 48h
+  steps:
+    cluster_profile: equinix-ocp-metal
+    workflow: baremetalds-e2e-ovn-ipv4
```
Comment (dgoodwin) [inline ci-operator/config/openshift/release/openshift-release-master__ci-4.11.yaml:237]: An additional job per 48h hopefully won't hit our quota too hard. Do you want to do one for 4.12 as well?

**Comment by DennisPeriquet** [comment]
looks like a `make jobs` is required.

File: ci-operator/config/openshift/release/openshift-release-master__ci-4.11.yaml (hunk 1 above)
Comment (jluhrsen) [reply ci-operator/config/openshift/release/openshift-release-master__ci-4.11.yaml:237]: yeah, good idea. PR updated with that. I think 4.12 wasn't even around when I originally made this change back in May.

**Comment by jluhrsen** [comment]
/hold
this got updated improperly somehow. will have to clean it up.

**Comment by jluhrsen** [comment]
> /hold this got updated improperly somehow. will have to clean it up.

/hold cancel

**Comment by jluhrsen** [comment]
> looks like a `make jobs` is required.

I think it's good now

**Comment by DennisPeriquet** [comment]
@jluhrsen Devan is out this week, but let me bring this up with the the TRT team so we can get a `/lgtm` on it.  It shouldn't be a problem as I figure, worst case, if badness ensues, we can always revert and tweak as needed.

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**